angr/storage/paged_memory.py

```diff
@@ -490,11 +489,19 @@ def load_objects(self, addr, num_bytes, ret_on_segv=False):
     #
 
     def _create_page(self, page_num, permissions=None):
-        return Page(
+        if self.state:
```

Comment:
Nitpick: Explicit is better than implicit -- I prefer `if self.state is not None` here.